ci-operator/config/openshift-pipelines/task-buildpacks/openshift-pipelines-task-buildpacks-main.yaml

```
@@ -35,8 +35,8 @@ tests:
   cluster_claim:
```

```suggestion
  cluster: build05
  cluster_claim:
```
And run `make jobs` to update the test point correct cluster